webapp/sources/rudder/rudder-core/src/test/resources/hooks.d/test/timeout_ok.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# this "HOOK_WARN_TIMEOUT" will not be used, neither the following, because of quotes:
+
+# "HOOK_WARN_TIMEOUT" = 1ms
+
+# HOOK_WARN_TIMEOUT = 6 s
+# HOOK_KILL_TIMEOUT = 10 s
+
+sleep 2
+
+exit 0
```

Comment:
could you also add a test for custom hook timeout time defined ?

Reply:
this is the case here, the test code says "kill after one second", but here the hook says "actually, kill after 10s", and since I sleep 2s, it tests that it's the custom timeout that is used.

Reply:
HA ! I thought it would not be a comment to b usable
This is really surprising - but actually it makes sense as in some language it wouldn't make sense. So it means I didn't read the explaination carefully